<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainWindow</class>
 <widget class="QMainWindow" name="ClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientMainWindow</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow { background-color: #f0fdf4; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_plant">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="button_back3">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
              </property>
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>218</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_myplant">
                <property name="styleSheet">
                 <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
                </property>
                <property name="text">
                 <string>내 식물:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_myplantname">
                <property name="styleSheet">
                 <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
                </property>
                <property name="text">
                 <string>이름</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
                </property>
                <property name="text">
                 <string>온도</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="styleSheet">
               <string>QProgressBar { border: 1px solid #cfcfcf; border-radius: 8px; text-align: center; }QProgressBar::chunk { background-color: #f48fb1; border-radius: 8px; }</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
                </property>
                <property name="text">
                 <string>습도</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar_2">
              <property name="styleSheet">
               <string>QProgressBar { border: 1px solid #cfcfcf; border-radius: 8px; text-align: center; }QProgressBar::chunk { background-color: #81d4fa; border-radius: 8px; }</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_write_post">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="button_back2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>228</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLineEdit" name="input_title">
            <property name="text">
             <string>input title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_file">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
            </property>
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="image_plant">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
          </property>
          <property name="text">
           <string>plant image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="input_post">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>400</height>
           </size>
          </property>
          <property name="plainText">
           <string>input content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_save">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_gallery">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="button_myplant">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
              </property>
              <property name="text">
               <string>내식물</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_writepost">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
              </property>
              <property name="text">
               <string>글쓰기</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_show_post">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="button_back">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>QPushButton { background-color: #88c999; color: white; border-radius: 5px; padding: 5px; }</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="post_title">
            <property name="styleSheet">
             <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
            </property>
            <property name="text">
             <string>제목 표시할거임</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="post_author_id">
            <property name="styleSheet">
             <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
            </property>
            <property name="text">
             <string>작성자(아이디) 표시</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="post_image">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>QLabel { color: #2d6b2d; font-weight: bold; }</string>
          </property>
          <property name="text">
           <string>이미지띄울거임</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="post_content">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="chatting_room">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string extracomment="ㅁㄴㅇㄻ"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_chatting">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
